web_playground/README.md

```diff
@@ -0,0 +1,5 @@
+Web playground that allows you to compile re2c code in your browser and inspect a lexer graph.
```

Comment:
Can you mention how it works here?

Reply:
I have added a screenshot. Also changed the sentence a bit.
What kind of information you would like me to add? Is it needed for a user or for a maintainer?

Reply:
> I have added a screenshot. 

I'd rather actually remove the screenshot, as it's better to keep the docs in plain text format, so that they can be viewed with a simple text editor. Besides, it will get outdated as we add new features.

> What kind of information you would like me to add? Is it needed for a user or for a maintainer?

For developers and maintainers: just a short explanation how the playground works (in particular, how re2c.js is generated). I think you can remove the "Developers" header (as this whole README is mostly intended for developers).

Reply:
Юлия не спит, хомячки пытаються что-то запушить в репозиторий.
(I'm Dutch, but I know Russian)

> I'd rather actually remove the screenshot, as it's better to keep the docs in plain text format, so that they can be viewed with a simple text editor. Besides, it will get outdated as we add new features.

Yes, agreed. I will fix it soon.

Reply:
Всегда была хорошего мнения о хомяках, и не ошибалась. :D

Reply:
I have expanded the Readme and added several extra comments to the `index.html` file.
Just in case, `build.sh` is full of comments as well. That is why Readme is so short.

Reply:
Great, thanks! I saw the comments in build.sh, they are very helpful.